docs/usage/writing/writing-to-s3-with-locking-provider.md

```diff
@@ -1,7 +1,10 @@
 # Writing to S3 with a locking provider
 
-A locking mechanism is needed to prevent unsafe concurrent writes to a
-delta lake directory when writing to S3.
+Delta lake guarantees [ACID transactions](https://delta-io.github.io/delta-rs/how-delta-lake-works/delta-lake-acid-transactions/) when writing data. This is done by default when writing to all supported object stores except AWS S3. (Some S3 clients like CloudFlare R2 or MinIO may enable atomic renames, refer to [this section](#enabling-concurrent-writes-for-alternative-clients) for more information).
```

Comment:
I'd like to make this an internal link like sphinx's cross referencing, instead of just using the direct link, but I'm not sure what the correct syntax would be to point to the `delta-lake-acid-transactions` page from here